lib/house.rb

```diff
@@ -0,0 +1,36 @@
+class House
+
+  attr_reader :price, :address, :rooms
```

Comment:
Keep an eye on the interaction pattern - the price should have been a separate method that returned price as an integer (`400000` instead of `"$4000000"`)